projects/zabbix/inventory.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python
```

Comment:
This script will get overwritten by the bootstrap process. I suspect it is not needed.